public/app/features/dashboard/components/Inspector/InspectContent.tsx

```diff
@@ -52,8 +60,16 @@ export const InspectContent = ({
   }
 
   let errors = data?.errors;
-  if (!errors?.length && data?.error) {
-    errors = [data.error];
+  if (!errors?.length) {
+    if (data?.error) {
```

Comment:
🤔  I noticed we can have the old deprecated single `error` together with the new `errors` array. 

For example, when throwing an error with the Test Data Source  (conditional error), together with the transformation error.

In those cases, we are not showing all the errors. Maybe is a good idea to add an extra condition to see if we have both errors and merge them. 


![MergeErrors](https://github.com/grafana/grafana/assets/239999/64a1cbb4-f064-4a92-947e-729a65d4505a)

Reply:
I updated so it will show both...  
<img width="1182" alt="image" src="https://github.com/grafana/grafana/assets/705951/6f39b32a-5956-402b-b2b5-80e9f412c764">